### ISSUE TITLE
Error when trying to use less

### ISSUE THREAD

**Issue by morgoth** [middleman/middleman-sprockets#90]
I'm having `application.css.less` with simple `body {color: red;}`
I got error (using middleman-sprockets )
http://storage6.static.itmages.com/i/16/0219/h_1455902604_6509929_8a77d72b7d.png

**Comment by stevenosloan** [comment]
Thanks @morgoth, I'm having some trouble reproducing -- can you provide a bit more information (an example project would be ideal), but hoping this would be enough:
- Your Gemfile & Gemfile.lock
- Your config.rb
- Basic project structure (mainly which path application.css.less is at)

**Comment by morgoth** [comment]
I created fresh middleman project (4.1.1) by `middleman init` and the error is the same
Gemfile:
```
gem 'middleman', '>= 4.0.0'
gem 'less'
gem 'therubyracer'
gem "middleman-sprockets", github: "middleman/middleman-sprockets"
```
config.rb is not changed at all.
source/stylesheets/site.css.less
```
body {
  color: red;
}
```

**Comment by stevenosloan** [comment]
Awesome, thanks -- got that reproducing now (not sure why my initial test case wasn't catching it).  

Unfortunately it looks like sprockets has dropped less support with 3.x https://github.com/sstephenson/sprockets/pull/547. For most things I think the built in less support of middleman should be good -- and in general we shouldn't be using sprockets for assets it doesn't support.

Thanks for the report!

**Comment by stevenosloan** [comment]
@morgoth now that I've merged #93 this should be resolved, please re-open and let me know if you continue to see issues.
